README.md

```diff
@@ -44,17 +44,61 @@ normalization logic.
 
 ### Run & Stop
 
-        $ ./tempesta.sh start
-        $ ./tempesta.sh stop
+Use ```tempesta.sh``` script to run and stop Tempesta. The script provides help
```

Comment:
General note: a single opening backquote before text, and a single closing backquote after the text should be used when the goal is to emphasize inline code. Triple backquotes are for blocks of code.